spring-rabbit/src/test/java/org/springframework/amqp/rabbit/connection/ClientRecoveryCompatibilityTests.java

```diff
@@ -68,6 +77,12 @@ public void testDefeatRecovery() throws Exception {
 		when(rabbitConn.isOpen()).thenReturn(false).thenReturn(true);
 		when(channel1.isOpen()).thenReturn(false);
 		conn2 = ccf.createConnection();
+		try {
+			channel = conn2.createChannel(false);
```

Comment:
Assignment isn't used.
Plus looks like we need `fail()`  after this operator.
Otherwise in case of successful execution we don't step into the `catch` below.